app/pages/cif/projects.tsx

```diff
@@ -85,13 +93,15 @@ export function Projects({ preloadedQuery }: RelayProps<{}, projectsQuery>) {
         </Card>
       ))}
       <h1>Pending Projects</h1>
-      {query.allFormChanges.edges.length === 0 && <p>None</p>}
-      {query.allFormChanges.edges.map(({ node }) => {
-        const cardTitle = `${node.newFormData.rfpNumber} (${node.changeStatus})`;
+      {query.allProjectRevisions.edges.length === 0 && <p>None</p>}
+      {query.allProjectRevisions.edges.map(({ node }) => {
+        const projectChangeNode =
+          node.formChangesByProjectRevisionId.edges[0].node;
+        const cardTitle = `${projectChangeNode.newFormData.rfpNumber} (${projectChangeNode.changeStatus})`;
```

Comment:
This is going to be completely rewritten in #95

Reply:
definitely, this is just a placeholder for now